pyscf/adc/radc.py

```diff
@@ -442,6 +518,37 @@ def density_matrix(myadc, T=None):
     return dm
 
 
+def analyze(myadc):
+
+    logger.info(myadc, "\n*************************************************************")
+    logger.info(myadc, "                Eigenvector analysis summary")
+    logger.info(myadc, "*************************************************************")
```

Comment:
The parentheses are a bit confusing. You might consider making this a multi-line string

```
logger.info(myadc, '''
*************************************************************
                Eigenvector analysis summary
*************************************************************''')
```